pkg/accumulator/accumulator_interface.go

```diff
@@ -2,7 +2,7 @@ package accumulator
 
```

Comment:
a better naming convention for this file is to call it `interface.go` (it is already under the accumulator directory and package.